kw

```diff
@@ -91,7 +91,11 @@ if [[ "${KW_REPO_MODE}" == 'y' ]]; then
   repo_mode_msg+=$'\t'"KW_PLUGINS_DIR=${KW_PLUGINS_DIR}"$'\n'
   repo_mode_msg+=$'\t'"KW_SOUND_DIR=${KW_SOUND_DIR}"$'\n'$'\n'
   repo_mode_msg+="${SEPARATOR}"
-  say "${repo_mode_msg}"
+
+  # Message is redirected to stderr such that it can be ignored if needed.
+  # Otherwise, it could mess up the output expected by tests which are runnning
+  # kw in repo-mode and the tests would fail.
+  say "${repo_mode_msg}" >&2
```

Comment:
From what I suspect this is an issue just for the integration tests, right? If so, shouldn't this come with the integration test in case, or, at least stress it in the comment that this is for the integration tests specifically?

Reply:
Yes. Eventually, we may want to use KW in repo-mode during integration tests. For example, we may want to run KW without installing it. I'll add such comment.